src/main/frontend/src/components/aggregates/Route.svelte

```diff
@@ -15,8 +15,8 @@
 	export let id;
 
 	let requestMethodDisabled = false;
-	$: changedMethodOrMethods(aggregateMethod, methods);
```

Comment:
This is needed, otherwise it won't react to changes of `aggregateMethod`. See other comment v

Reply:
fixed